<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>474</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>474</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbMemory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Memory</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>43</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vlMemory">
      <item>
       <widget class="QLabel" name="lMem">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>( )</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMemClear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>MC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMemResult">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>MR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMemSave">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>MS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbMemPlus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>M+</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLineEdit" name="leNumber">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12345678901234567</string>
    </property>
    <property name="maxLength">
     <number>17</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>241</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="glMain">
     <item row="0" column="2">
      <widget class="QPushButton" name="pbBackspace">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>BS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pbCleanEntry">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pbClear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pb7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pb8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pb9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pbDiv">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pbSqrt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Sqrt</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pb4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pb5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pb6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pbMul">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pbReverse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>1/x</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pb1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pb2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pb3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pb0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pbAddSign">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pbSeparator">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Sep</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pbPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pbResult">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>311</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlComplex">
     <item>
      <widget class="QPushButton" name="pbPower">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Pwr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRoot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Root</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbModule">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Mdl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDeg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Cnd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRad">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Cnr</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>369</y>
      <width>300</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlFuncBoxes">
     <item>
      <widget class="QLineEdit" name="lePow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFunct">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>251</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>251</width>
         <height>41</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="options">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="saveFormat"/>
    <addaction name="History"/>
   </widget>
   <addaction name="options"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="saveFormat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сохранять формат</string>
   </property>
   <property name="whatsThis">
    <string>Сохранять разделитель при отображении результата с нулевым значение мнимой части</string>
   </property>
  </action>
  <action name="History">
   <property name="text">
    <string>История</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
